docs/how-to/code/internal-user/task.yaml

```diff
@@ -12,11 +12,13 @@ execute: |
   rockcraft pack
   # [docs:pack-rock-end]
 
+  # [docs:skopeo]
   sudo rockcraft.skopeo --insecure-policy copy oci-archive:internal-user_latest_amd64.rock docker-daemon:internal-user:latest
+  # [docs:skopeo-end]
 
   # [docs:check-user]
-  id=$(docker run -d internal-user:latest --verbose)
-  docker exec "$id" pebble logs | grep ubuntu
+  id=$(docker run -d -p 8080:8080 internal-user:latest --verbose)
+  sleep 5
+  docker logs "$id" 2> &1 | grep "Serving by myuser"
   docker rm -f "$id"
-  # [docs:check-user-end]
- 
\ No newline at end of file
+  # [docs:check-user-end]
```

Comment:
missing newline

Reply:
you previously requested that change [here](https://github.com/canonical/rockcraft/pull/653/#pullrequestreview-2216211213)

Reply:
ah yeah, that's a diff glitch I guess...I think I suggested removing empty spaces - you had "    ". But I wanted to preserver the newline. GH assumed line deletion though